libraries/AP_DDS/AP_DDS_Client.h

```diff
@@ -15,6 +15,7 @@
 #endif // AP_DDS_TIME_PUB_ENABLED
 #if AP_DDS_NAVSATFIX_PUB_ENABLED
 #include "sensor_msgs/msg/NavSatFix.h"
+#include <AP_GPS/AP_GPS.h>
```

Comment:
```suggestion
#include <AP_GPS/AP_GPS_config.h>
```

It's defined here: https://github.com/ArduPilot/ardupilot/blob/b648b5564051bf10b1f4d69fa004b158773cadd9/libraries/AP_GPS/AP_GPS_config.h#L19
This will reduce compilation time.